### ISSUE TITLE
Siblings #tags/sibling is not supported

### ISSUE THREAD

**Issue by dariuszkowalski-com** [OfficerHalf/obsidian-auto-class#28]
I would like to add CSS to all files with a tags:

`#📆/DayNote/`

where  I can have  `#📆/DayNote/Saturday` , ` #📆/DayNote/Monday `etc...

Now it doesn't work. I need to add all combination of tags.

**Comment by thorlucas** [comment]
Yep, this is a significant problem especially because it breaks semantics of what tags *are*. Something tagged `#foo/bar` _is automatically also_ tagged `#foo`.

**Comment by OfficerHalf** [comment]
Yeah, I think this is probably the biggest gap in functionality currently. I'll see about adding this soon.
